src/modules/schedule/typedefs/schedule.ts

```diff
@@ -0,0 +1,87 @@
+import { gql } from "graphql-tag";
+
+const typedef = gql`
+  type Schedule {
+    """
+    The ObjectID associated with the schedule record
+    """
+    _id: ID
+    """
+    The name of the schedule, such as "Oski's Fall schedule <3"
+    """
+    name: String
+    """
+    Identifier (probably email) for the user who created the schedule (such as oski@bereley.edu).
+    """
+    created_by: String!
+    """
+    Term corresponding to the schedule, such as "Fall 1986"
+    """
+    term: String!
+    """
+    Whether the user would like the schedule to be viewable by others.
+    """
+    public: Boolean!
+    """
+    Identifiers (probably cs-course-ids) for the classes the user has added to their schedule.
+    """
+    class_IDs: [String]
```

Comment:
`[String!]` ensures that all entries of array are non-null strings—should be useful for most `[String]`